Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.cs

```diff
@@ -10,59 +10,48 @@
 // THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
 // ******************************************************************
 
-using System.Numerics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
 using Windows.ApplicationModel;
 using Windows.Foundation.Metadata;
-using Windows.UI;
 using Windows.UI.Composition;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Documents;
 using Windows.UI.Xaml.Hosting;
+using Windows.UI.Xaml.Input;
 using Windows.UI.Xaml.Markup;
-using Windows.UI.Xaml.Shapes;
+using Windows.UI.Xaml.Media;
 
 namespace Microsoft.Toolkit.Uwp.UI.Controls
 {
     /// <summary>
     /// The <see cref="DropShadowPanel"/> control allows the creation of a DropShadow for any Xaml FrameworkElement in markup
     /// making it easier to add shadows to Xaml without having to directly drop down to Windows.UI.Composition APIs.
     /// </summary>
-    [ContentProperty(Name = nameof(CastingElement))]
-    public partial class DropShadowPanel : UserControl
+    [TemplatePart(Name = PartShadow, Type = typeof(Border))]
+    [TemplatePart(Name = PartContent, Type = typeof(ContentPresenter))]
+    [ContentProperty(Name = PartContent)]
+    public partial class DropShadowPanel : Control
```

Comment:
@skendrot the dropshadow is generated on a border that precedes the content requesting dropshadow. It has under it in visual tree. Look at the xaml